src/components/utils/table.tsx

```diff
@@ -320,7 +320,7 @@ export const filterRowsByIndexes = (
     const relativeIndexes = absoluteIndexes.reduce<number[]>((indexes, absoluteIndex) => {
       const { parentIndex, index } = absoluteIndexesMap[absoluteIndex] || defaultAbsoluteIndex;
       if (index >= 0) {
-        if (parentIndex === rootIndex) {
+        if (parentIndex === rootIndex && rows[index]) {
```

Comment:
do we need this check. The two data structures must be compatible. Otherwise, there's a risk of data inconsistency. In this case, I prefer an error to alert, rather than introducing an inconsistency. WDT?

Reply:
Fixed with https://github.com/PricemoovOSS/react-table/pull/6#discussion_r1339871033